### DISCUSSION TITLE
add underscore in citation key

### DISCUSSION THREAD

**Discussion by cdmcdermott** [retorquere/zotero-better-bibtex#2195] [Q&A] [answered]
I am using the following citation key format:   auth.lower + shorttitle(1,1).lower + year

This produces a key like browntowards2019

How can I included '_' in the key so it apepars as brown_towards_2019

**Comment by cdmcdermott** [comment, marked as answer]
auth.lower.postfix(\_) + shorttitle(1,1).lower.postfix(\_) + year

**Comment by thistlillo** [comment]
My current citation key format is:
```
authEtAl2.lower +year+veryshorttitle.lower
```
If I try and add `postfix()` as it follows:
```
authEtAl2.lower.postfix() +year+veryshorttitle.lower
```
BetterBibtex refuses it: after ok the format is turned back into the first one.

What did I not get?

**Comment by retorquere** [comment]
`postfix` needs a parameter. If you hover over the field when it is orange you get the error message.

**Comment by thistlillo** [comment]
Thanks, I did not read (my mistake!) the documentation but only the question and the answer in this issue.

**Comment by retorquere** [comment]
Oh you're right,the answer above didn't show correctly. There was an underscore there, but markdown turns that into italics. Fixed now.
